themes/default/content/blog/multi-lang-hackathon/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Demos from the Multi-Language Component Hackathon"
+date: 2021-06-24
+meta_desc: "Pulumi successful multi-lang package hackathon showed how to build multi-lang components and yielded demos exploring new concepts and capabcilities"
+meta_image: hackathon.png
+authors:
+    - lee-zen
+tags:
+    - multi-language component
+    - hackathon
+---
+
+At Pulumi, we have a tradition of hosting hackathons every so often to play with concepts and ideas that we may not typically encounter in our day-to-day product building activities. This past week, we’ve had two separate back-to-back hackathons. Our first hackathon, which was open to the community, focused on using our new [multi-language component capabilities]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}). Multi-language components allow developers to author reusable infrastructure abstractions in one language and make them available to others in all the languages that Pulumi supports. We were really excited to see what everyone would build!
+
+<!--more-->
+
+We had a kickoff meeting on Tuesday morning where Lee Briggs walked participants through the [basic steps for bringing up a multi-language component]("https://www.youtube.com/watch?v=_RXvNS5N8A8). If you’re interested in seeing the kickoff meeting, we’ve recorded it and have made it available for viewing on [Pulumi TV](https://youtu.be/Ogg5cs6vPfc). After the walk-through, teams got down to business!
+
+We had some great ideas, including but not limited to components for a simplified Google Cloud project setup, an Azure web app service container, and an Amazon SageMaker model with a frontend for serving it.
+
+As part of the hackathon, we received some great feedback from all the participants on improving the multi-language authoring experience, including reducing boilerplate code, ways to improve on the schema authoring experience, and documentation suggestions. We’re keen to continue to refine the experience.
+
+Thank you to all the participants! If you’d like to see some of the components in action, you can [watch our demo session](https://youtu.be/U7-eYSpB4o8). We saw the following component demos:
```

Comment:
Did we get any from the community? If so we should draw attention to those

Reply:
Not aware of any, any one @leezen?

Reply:
I believe the demo Paul had of GCP project scaffolding was done in conjunction with Simen.